docs/products/clickhouse/concepts/databases-and-tables.rst

```diff
@@ -10,23 +10,21 @@ Databases and tables are at the core of any **Database Management System**. Clic
 Databases
 ---------
 
-Similarly when working with any other database technology, the first step is to create the database itself. Due to security reasoning, the `admin` user is not allowed to create databases. However, you can create a new database through the web interface of `Aiven console <https://console.aiven.io/>`_:
+Creating tables in an Aiven for ClickHouse service can only be done via the Aiven platform; the `admin` user is not allowed to create databases directly for security and reliability reasons. However, you can create a new database through the web interface of `Aiven console <https://console.aiven.io/>`_:
 
 #. Log in to `Aiven console <https://console.aiven.io/>`_ and select your service.
 #. On the service page open the tab *Databases & Tables*.
 #. Enter your database name in *Create a new database* and click on **Create database**.
 #. On our side we enable necessary customizations and run secondary queries to grant access to the admin user.
 
-Now you can see your database in the *Database list* below.
-
 .. note::
 
-    If you try adding a new database to **Aiven for ClickHouse** through the command line, you'll encounter an exception ``Not enough privileges.(ACCESS_DENIED)``. Please use the web interface to add a new database.
+    If you try adding a new database to **Aiven for ClickHouse** through the command line, you'll encounter an exception ``Not enough privileges.(ACCESS_DENIED)``. Please use the Aiven platform, either the web interface or one of our other tools, to add a new database.
```

Comment:
" or one of our other tools" - what are the other tools?

Reply:
THERE it is, I knew I'd edited this in another place and I couldn't locate it with keywords like "API". Thanks!